etl/android-clients.kp/knowledge.md

```diff
@@ -161,11 +161,11 @@ while day <= end:
             StructField("defaultsearch", StringType(), True),
             StructField("device", StringType(), True),
             StructField("arch", StringType(), True),
-            StructField("fennecActivityStreamTopsitesLoaderTimeMs", 
+            StructField("fennec_activity_stream_topsites_loader_time_ms", 
```

Comment:
Ok, I can live with minimal change trumping full consistency in naming. Per IRC conversation, the direct-to-parquet output should replace this dataset eventually anyways.